src/main/java/com/helpchat/redisbungeeexpansion/RedisBungeeExpansion.java

```diff
@@ -55,8 +54,23 @@ public boolean register() {
                 servers.put(s, 0);
             }
         }
+        boolean registered;
+        try {
+            //for older version of papi support.
```

Comment:
I'd honestly say drop support for older versions of PAPI. They don't receive support from us anymore. Compile against latest, ensure it works on latest, don't worry about doing hacks to make it work on previous versions.